_posts/2019-02-12-funding-eslint-future.md

```diff
@@ -0,0 +1,66 @@
+* **Rewarding contributions from the community.** We'd like to reward everyone who contributes to ESLint, whether it's a one-time contribution or ongoing contributions. We are still figuring out the details, but rewarding people with everything from stickers and t-shirts to cash payouts for significant contributions are things we are thinking about right now.
+* **Improve documentation.** ESLint's documentation hasn't had a significant overhaul since the project was first released. We feel there are a lot of ways to improve the documentation, including potentially contracting an information architect or technical writer to improve our documentation.
+* **Better communication and support.* By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
+* **Developing a Roadmap.** For a while, the ESLint team has mostly worked on a one-off basis in terms of which features to implement. Without knowing who would be able to spend what amount of time on the project, it was difficult to do long-term planning and develop a roadmap for the future. This is something we will finally be able to do when we are a self-sustaining project.
+
+## Our Goal: $20,000 per month
+
+In order to achieve everything we have in mind, we believe that we will need $20,000 per month. With this amount, we can sustain ESLint for the foreseeable future.
+
+## Introducing our first sponsors
+
+<p class="text-center">
+    <a href="https://facebook.com"><img src="/img/logos/facebook.png" alt="Facebook" height="96"></a>
+    <a href="https://airbnb.com"><img src="/img/logos/airbnb.png" alt="Airbnb" height="96"></a><br>
+    <a href="https://frontendmasters.com"><img src="https://static.frontendmasters.com/assets/fm/js/frontendmasters.0e71088726.svg" alt="Frontend Masters" height="32"></a>
+</p>
+
+We're also excited to announce our first sponsors. [Facebook](https://facebook.com) and [Airbnb](https://airbnb.com) have both pledged $1,000 per month to support ESLint.
+
+Additionally, [Frontend Masters](https://frontendmasters.com) has pledged $200 per month to support ESLint.
+
+If your company also uses ESLint to find and fix problems in your JavaScript code, please ask them to join these fantastic companies in [sponsoring ESLint](https://opencollective.com/eslint). (All sponsors above $200 per month get their logos on both the ESLint README and on the homepage of eslint.org.)
+
+We'd like to thank Eric Nakagawa at Facebook, Jordan Harband at Airbnb, Marc Grabanski at Frontend Masters, and Pia Mancini at Open Collective for supporting our early sponsorship efforts at their respective companies.
```

Comment:
What do you think about linking Twitter or GitHub profiles for these people?

Reply:
I don't think its necessary. All of them were working to represent their respective companies, and we already are linking directly to those companies.